bitcoin/src/network/constants.rs

```diff
@@ -582,6 +600,7 @@ mod tests {
         }
     }
 
+
```

Comment:
Nit: you still added a newline here. (This is fine, I won't hold things up over it)

Reply:
Oops, if i were to go in and change this, would I have to re-squash and then get another approval? or would it be fine to merge with 2 commits